packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -187,6 +189,12 @@ class CupertinoPicker extends StatefulWidget {
   /// Defaults to `1.45` to visually mimic iOS.
   final double squeeze;
 
+  /// The behavior of reporting the selected item index.
+  ///
+  /// This determines when the `onSelectedItemChanged` callback is called.
+  /// Defaults to [ChangeReportingBehavior.onScrollUpdate].
+  final ChangeReportingBehavior changeReportingBehavior;
```

Comment:
We should mention what the default native behavior is, and the current version of iOS.